....E...F.E...F......E...E..F..F.........
Time: 0.082
There were 4 errors:
1) test30(RandoopTest0)java.lang.NullPointerException
	at RandoopTest0.test30(RandoopTest0.java:535)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
2) test26(RandoopTest0)java.lang.NullPointerException
	at RandoopTest0.test26(RandoopTest0.java:463)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
3) test17(RandoopTest0)java.lang.NullPointerException
	at RandoopTest0.test17(RandoopTest0.java:305)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
4) test14(RandoopTest0)java.lang.NullPointerException
	at RandoopTest0.test14(RandoopTest0.java:246)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
There were 4 failures:
1) test27(RandoopTest0)junit.framework.AssertionFailedError: 'java.security.Permissions@2a5ebf99 (
)
' != 'java.security.Permissions@592c34b (
)
'
	at RandoopTest0.test27(RandoopTest0.java:492)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
2) test23(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test23(RandoopTest0.java:411)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
3) test12(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test12(RandoopTest0.java:214)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
4) test10(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test10(RandoopTest0.java:167)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)

FAILURES!!!
Tests run: 33,  Failures: 4,  Errors: 4

Cycle length is4
before
after
Reached this
# of deadlocks detected 0
THE SIZE OF THE TRACE OMEN 34678
Exiting analysis
